src/components/Card/SmartphoneCard.tsx

```diff
@@ -0,0 +1,134 @@
+import { FC } from 'react';
+import { Rate } from 'antd';
+import styles from './card.module.css';
+import { rateImg, rateEmptyImg } from '../../assets/constants';
+import { Link } from 'react-router-dom';
+import useProductCardHandlers from '../../hooks/useProductCardHandlers';
+import { IAccessory } from '@/interfaces/interfaces';
+
+interface ISmartphoneCard {
+  product: IAccessory;
+  classname: string;
+}
+
+const SmartphoneCard: FC<ISmartphoneCard> = ({ product, classname }) => {
+  const {
+    isHovered,
+    isLiked,
+    isMouseDown,
+    handleClickBuy,
+    handleMouseDown,
+    handleMouseUp,
+    handleMouseLeave,
+    handleMouseEnter,
+    handleClickLike,
+  } = useProductCardHandlers();
+
+  const accessoryProduct = product as IAccessory;
+  const productRating = accessoryProduct.rate ?? 0;
+
+  return (
+    <>
+      <Link
+        className={`${styles.cardConatiner} ${classname} `}
+        key={accessoryProduct?.id}
+        to="/smartphone/apple iPhone 15 Pro 256Gb Blue Titanium"
+      >
+        <div
```

Comment:
lets avoid using style in the jsx. We can use classnames and create a styles for the conditions